assets/components/lvms/topolvm-configmap_lvms-version.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: lvms-version
+  namespace: openshift-storage
```

Comment:
The microshift-version config map goes into kube-public to ensure special RBAC rules aren't needed to read it. Will an agent using a service account need special permission to read data from this namespace?

Reply:
AFAICT, new and existing SAs at least have view access to CMs across the cluster.  Tested this by creating a namespace and using it's default SA to read the CM in the openshift-storage namespace.